app/javascript/styles/application.scss

```diff
@@ -1,28 +1,29 @@
-@import 'mastodon/mixins';
-@import 'mastodon/variables';
-@import 'fonts/roboto';
-@import 'fonts/roboto-mono';
-@import 'fonts/montserrat';
+@import "mastodon/mixins";
+@import "mastodon/variables";
+@import "fonts/roboto";
+@import "fonts/roboto-mono";
+@import "fonts/montserrat";
+@import "fonts/phosphate";
```

Comment:
Would be nice to avoid changing files in the core system like this. I'm guessing your code editor has applied some automate styles. Please remove these changes :)